cmd/theatre-envconsul/main.go

```diff
@@ -133,6 +134,37 @@ func mainError(ctx context.Context, command string) (err error) {
 			}
 		}
 
+		var filePaths = environment{}
+		const pathTBD = ""
+
+		// Rewrite vault-file: envs to vault: envs. Store the paths to which they should
```

Comment:
Would be good in this comment to add some examples of what it actually looks like, for clarity

e.g.
`vault-file:tls-key/2021010100`
`vault-file:ssh-key/2021010100:/home/user/.ssh/id_rsa`